routes/index.js

```diff
@@ -75,6 +75,11 @@ router.get('/overlays/springshowdown/all4by3', function(req, res) {
   res.render('overlays/springshowdown/all4by3', { title: 'Spring Showdown Overlay', layout: false });
 });
 
+/*Generic*/
+router.get('/overlays/generic/generic', function(req, res) {
+  res.render('overlays/generic/generic', { title: 'Generic Overlay', layout: false });
```

Comment:
This organizational structure is a little questionable. I think the route should just be

`overlays/generic`

Unless there are other views in the generic folder, there's no reason to have a 3 deep structure.